src/Aspire.Hosting/Pipelines/IPipelineOutputService.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Pipelines;
+
+/// <summary>
+/// Service for managing pipeline output directories.
+/// </summary>
+[Experimental("ASPIREPIPELINES001", UrlFormat = "https://aka.ms/aspire/diagnostics/{0}")]
```

Comment:
```suggestion
[Experimental("ASPIREPIPELINES004", UrlFormat = "https://aka.ms/aspire/diagnostics/{0}")]
```

Let's create a new code for this to stop overloading on the code for the core APIs.

Reply:
@copilot Update the diagnostic ID introduced in this Experimental attribute.